docs/trace_service_features.md

```diff
@@ -2,34 +2,38 @@
 
 ## TOC
 
-- [Ad Service](#ad-service)
-- [Cart Service](#cart-service)
-- [Checkout Service](#checkout-service)
-- [Currency Service](#currency-service)
-- [Email Service](#email-service)
-- [Feature Flag Service](#feature-flag-service)
-- [Frontend Service](#frontend-service)
-- [Payment Service](#payment-service)
-- [Product Catalog Service](#product-catalog-service)
-- [Recommendation Service](#recommendation-service)
-- [Shipping Service](#shipping-service)
-
-## OpenTelemetry Feature Set by Services
+- [C++](#c)
```

Comment:
c -> c++ ?

Reply:
C++ is a broken link. It needs to be c for it work